proc_scripts/phasing.py

```diff
@@ -92,13 +95,61 @@ def zeroth_order_ph(d, fl=None):
         ax.set_aspect('equal', adjustable='box')
         ax.add_patch(ell)
     return exp(1j*ph0)
-def hermitian_function_test(s, down_from_max=0.5, shift_val=1.0):
+
+def phasecorrect(d):
+    fl.push_marker() 
```

Comment:
what does this do?  I don't recognize the function name, and there is no doc string

Reply:
This is the first order phasing we implemented into my manual phasing 90 pulse experiments

Reply:
OK -- in that case:

- `phasecorrect` is an incredibly general name -- if I understand what correctly `phasecorrect_real_abs` or just `ph1_real_abs` might be a better name
- you need a docstring that explains what it does and gives input parameters as well as return values

Reply:
really rough description but a start. Will expand as we incorporate it into scripts